.travis.yml

```diff
@@ -26,6 +26,8 @@ before_script:
   - java -jar ./target/cargo-daemon-webapp-${CARGO_DAEMON_WEBAPP_VERSION}.war &
 
 script: 
+  - mvn xml:check-format
+  - mvn formatter:validate
```

Comment:
Formatter-maven-plugin checks format for all java files under the directory where command(mvn formatter:validate) executed by executing one command .
Therefore, there is no problem at present-setting .
On the other hand, since xml - maven - plugin performs format check on the project set in module, the format check is not executed by initdb and selenium in the current setting.
So I fixed it with the 5e876da7050d0f849184f48ce7203faa77d61635 commit.